src/pykeen/datasets/pharmkg8k.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+
+"""The `PharmKG-8k <https://github.com/biomed-AI/PharmKG/>`_ dataset.
+
+Get a summary with ``python -m pykeen.datasets.pharmkg8k``.
+"""
+
+import click
+from docdata import parse_docdata
+from more_click import verbose_option
+
+from .base import UnpackedRemoteDataset
+
+__all__ = [
+    "PharmKG8k",
+]
+
+BASE_URL = "https://raw.githubusercontent.com/biomed-AI/PharmKG/master/data/PharmKG-8k/"
+VALID_URL = f"{BASE_URL}/valid.tsv"
+TEST_URL = f"{BASE_URL}/test.tsv"
+TRAIN_URL = f"{BASE_URL}/train.tsv"
+
+
+@parse_docdata
+class PharmKG8k(UnpackedRemoteDataset):
+    """The PharmKG8k dataset from [zheng2020]_.
+
+    ---
+    name: PharmKG8k
```

Comment:
is the desired stylization? it doesn't match the title of this PR which reads `PharmKG-8k`

Reply:
Hmmmm seems the original authors style as `PharmKG-8k` so i'll style as such here.

Reply:
I decided to just go with `PharmKG8k` in the end. It also matches the formatting for the other dataset names in PyKEEN.